assets/component-newsletter.css

```diff
@@ -41,6 +45,16 @@
 .newsletter-form__button {
   width: 4.4rem;
   margin: 0;
+  right: var(--inputs-border-width);
+  top: 0;
+  height: 100%;
+}
+
+.newsletter-form__button:focus-visible,
+.newsletter-form__button:focus {
+  background: rgb(var(--color-background));
```

Comment:
I don't think safari is getting the same background fix 
<img src="https://screenshot.click/13-09-t7xr0-r85qh.png" width="350"/>